src/pages/quiz.js

```diff
@@ -19,22 +19,20 @@ export const quiz = (qNumber) => {
       text,
     })
   );
-
   // Generate new HTML
   const quizTemplate = createQuestion(
     currentQuestionText,
     currentAnswersData,
     pathname
   );
-
   // Add HTML to app
   document.getElementById('app').innerHTML = quizTemplate;
 
   document.addEventListener('click', (event) => {
-    if (event.target?.id === ANSWER_LABEL_ID) {
+    if (event.target.id === ANSWER_LABEL_ID) {
```

Comment:
```suggestion
    if (event.target?.id === ANSWER_LABEL_ID) {
```
Hi George. This code is not wrong, it's shorthand for:

` if (event.target && event.target.id === ANSWER_LABEL_ID)`